src/index.html

```diff
@@ -22,6 +22,362 @@
+          alt="Moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#Laptops $ Computer"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Tablet"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div
+          data-qa="card"
+          class="product-card"
+        >
+          <img
+            class="product-card__img"
+            src="/src/images/imac.jpeg"
+            alt="imac"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+          <div class="product-card__review">
+            <div class="product-card__stars">
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+            </div>
+            <p class="product-card__reviews">Reviews: 5</p>
+          </div>
+          <div class="product-card__price">
+            <span class="product-card__price-label">Price:</span>
+            <span class="product-card__price-value">$2,199</span>
+          </div>
+          <button
+            data-qa="card-hover"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
```

Comment:
The text inside the button should be in a span or a p tag for better accessibility.